docs/contributors.rst

```diff
@@ -339,31 +339,4 @@ Verify that the latest version of Horovod is now available:
 
     $ pip install --upgrade horovod
 
-
```

Comment:
no replacement here? The content of `docker/README.md`?

Reply:
Yeah, because this is done automatically through a workflow, we no longer need to perform these steps manually.

Reply:
ah, you mean this were the docker release steps, I was always wondering where I could find them...

Reply:
:)